test/optimus/pr/ValidateOptimus.wdl

```diff
@@ -69,21 +69,24 @@ task ValidateBam {
         File bam
         String expected_checksum
     }
-    
-    Int required_disk = ceil(size(bam, "G") * 1.1)
+
+Int required_disk = ceil(size(bam, "G") * 1.1)
 
     command <<<
+        cacheInvalidationRandomString=4
```

Comment:
Do we need to check these cache invalidating inputs in to the repo? seems like s temporary testing thing that can be removed (for each task)

Reply:
I found that it was essential to have something to change and I think it will be needed over and over again.